tooling/util/proxies.js

```diff
@@ -0,0 +1,178 @@
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+const crypto = require('crypto');
+const http = require('http');
+const path = require('path');
+const url = require('url');
+const yakbak = require('yakbak');
+
+/**
+ * A Cisco web service that we communicate with.
+ * @typedef {Object} Service
```

Comment:
@adamweeks does this work?

Reply:
Awesome, yea! 👍

Reply:
![img](https://media0.giphy.com/media/xUPGcxpCV81ebKh7Vu/giphy.gif?cid=e1bb72ff5bb62f554f4f525551817086)